src/figuresClasses.ts

```diff
@@ -1,19 +1,100 @@
-export interface Figure {
+type Shapes = 'triangle'|'circle'|'rectangle';
+type Colors = 'red'|'green'|'blue';
```

Comment:
```suggestion
type Shape = 'triangle' | 'circle' | 'rectangle';
type Color = 'red' | 'green' | 'blue';
```
use type names in singular form